src/ts/options.ts

```diff
@@ -72,6 +78,17 @@ $(function () {
     getOptions();
 });
 
+export function parseOptions (options: string) {
+    return Object.assign({}, defaultOptions, JSON.parse(options))
```

Comment:
I added this because the extension wasn't actually able to handle new options before when a user upgraded. This makes it so that new defaults are always merged with the currently saved options on a users machine when the extension starts.